tests/test_config.py

```diff
@@ -19,31 +19,36 @@
 plugins:
   generators:
     huggingface:
-      dtype: general
-      gpu: 0
+      hf_args:
+        torch_dtype: float16
       Pipeline:
-        dtype: bfloat16
+        hf_args:
+            device: cuda
   probes:
     test:
       generators:
         huggingface:
             Pipeline:
-              dtype: for_probe
+                hf_args:
+                    torch_dtype: float16
   detector:
       test:
         val: tests
         Blank:
           generators:
             huggingface:
-                gpu: 1
+                hf_args:
+                    torch_dtype: float16
+                    device: cuda:1
                 Pipeline:
                   dtype: for_detector
   buffs:
       test:
         Blank:
           generators:
             huggingface:
-                gpu: 1
+                hf_args:
+                    device: cuda:0
```

Comment:
...and `gpu: 1` changed to `cuda:0` here?

Reply:
`gpu` was a theoretical value `device` aligns with actual huggingface expectations.

Reply:
got it. demonstrating variation is good. otoh I think the example could be confusing with `gpu: 1` referring to both `cuda: 0` and `cuda: 1` within sight of each other.

Reply:
The final result of the edits is removal of all `gpu:` entries and only provides `huggingface` examples using the `device` key.